db/seeds.rb

```diff
@@ -1,24 +1,26 @@
 require "faker"
 require "csv"
 
-return unless Rails.env.in?(%w[development review separation])
-
-PaperTrail.enabled = false
-
-Faker::Config.locale = "en-GB"
-
 def load_base_file(file)
   base_file = Rails.root.join("db", "seeds", "base", file)
 
   load(base_file)
 end
 
+Rails.logger.info("seeding add_courses.rb")
+load_base_file("add_courses.rb")
```

Comment:
a bit confused, why are we moving this before the env checks? also might be safer to at least have a prod check?